File: src/Usuario.py
class Usuario:
    def __init__(self, nombre, apellido, direccion):
        self.__nombre = nombre
        self.__apellido = apellido
        self.__direccion = direccion
        self.__libros_rentados = []
        self.__libros_comprados = []

    def get_nombre(self):
        return self.__nombre

    def get_apellido(self):
        return self.__apellido

    def get_direccion(self):
        return self.__direccion

    def get_libros_rentados(self):
        return self.__libros_rentados

    def get_libros_comprados(self):
        return self.__libros_comprados

File: src/Usuario.java
import java.util.ArrayList;

public class Usuario {
    private String nombre;
    private String apellido;

    private String direccion;


    private ArrayList<Libro> librosRentados=new ArrayList<>();

    private ArrayList<Libro> librosComprados=new ArrayList<>();

    public String getNombre() {
        return nombre;
    }
    public String getApellido() {
        return apellido;
    }
    public String getDireccion() {
        return direccion;
    }
    public ArrayList<Libro> getLibrosRentados() {
        return librosRentados;
    }
    public ArrayList<Libro> getLibrosComprados() {
        return librosComprados;
    }

    public Usuario(String nombre,String apellido,String direccion) {
        this.nombre=nombre;
        this.apellido=apellido;
        this.direccion=direccion;
    }
}

